packages/core/src/Modifiers.ts

```diff
@@ -121,6 +121,12 @@ export class Modifiers {
         }
         return found;
     }
+    /**
+     * Return all the modifiers instances that are hold by `this`.
+     */
+    getContent(): Modifier[] {
+        return this._contents;
+    }
```

Comment:
No! The whole point of not using an array is to not manipulate `_contents` directly! If we wanted to be able to get it directly we wouldn't have made it private...